index.bs

```diff
@@ -5606,9 +5613,84 @@ A <dfn>geolocation override</dfn> is a [=struct=] with:
+
+The <dfn export for=commands>emulation.setForcedColorsModeThemeOverride</dfn> command modifies
+[=forced colors mode=] theming characteristics on the given top-level traversables or user contexts.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+    <pre class="cddl remote-cddl">
+      emulation.SetForcedColorsModeThemeOverride = (
+        method: "emulation.setForcedColorsModeThemeOverride",
+        params: emulation.SetForcedColorsModeThemeOverrideParameters
+      )
+
+      emulation.SetForcedColorsModeThemeOverrideParameters = {
+        theme: [+emulation.ForcedColorsModeAutomationTheme],
+        ? contexts: [+browsingContext.BrowsingContext],
+        ? userContexts: [+browser.UserContext],
+      }
+
+      emulation.ForcedColorsModeAutomationTheme = "none" / "light" / "dark"
+    </pre>
+   </dd>
+   <dt>Result Type</dt>
+   <dd>
+    <code>
+     EmptyResult
+    </code>
+   </dd>
+</dl>
+
+<div algorithm="remote end steps for emulation.setForcedColorsModeThemeOverride">
+
+The [=remote end steps=] with |command parameters| are:
+
+1. If |command parameters| doesn't [=map/contain=] "<code>theme</code>",
+   return [=error=] with [=error code=] [=invalid argument=].
+
+1. If |command parameters| [=map/contains=] "<code>theme</code>",
+   and "<code>theme</code>" is not "<code>none</code>", "<code>light</code>", or "<code>dark</code>",
+   return [=error=] with [=error code=] [=invalid argument=].
+1. If |command parameters| [=map/contains=] "<code>userContexts</code>"
+   and |command parameters| [=map/contains=] "<code>context</code>",
+   return [=error=] with [=error code=] [=invalid argument=].
+
+1. If |command parameters| doesn't [=map/contain=] "<code>userContexts</code>"
+   and |command parameters| doesn't [=map/contain=] "<code>context</code>",
+   return [=error=] with [=error code=] [=invalid argument=].
+
+1. Let |navigables| be a [=/set=].
+
+1. If the <code>contexts</code> field of |command parameters| is present:
+
+   1. Let |navigables| be the result of [=trying=] to [=get valid top-level traversables by ids=] with |command parameters|["<code>contexts</code>"].
+
+1. Otherwise, if the <code>userContexts</code> field of |command parameters| is present:
+
+   1. Let |user contexts| be the result of [=trying=] to [=get valid user contexts=] with |command parameters|["<code>userContexts</code>"].
+
+   1. For each |user context| of |user contexts|:
+
+      1. [=map/Set=] [=forced colors mode theme overrides map=][|user context|] to |command parameters|["<code>theme</code>"].
+
+      1. [=list/For each=] |top-level traversable| of the list of all [=/top-level traversables=]
+         whose [=associated user context=] is |user context|:
+
+         1. [=list/Append=] |top-level traversable| to |navigables|.
+
+1. For each |navigable| of |navigables|:
+
+   1. [=Set emulated forced colors theme data=] with |navigable| and |command parameters|["<code>theme</code>"].
```

Comment:
As a note for reviewers, this is dependent on https://github.com/w3c/csswg-drafts/pull/11823

Reply:
This has now landed.